web/docs/tutorials/todo-app/dependencies.md

```diff
@@ -21,11 +21,11 @@ app TodoApp {
 }
 ```
 
-Run (if it is already running, stop it first and then run it again)
+Run
 ```shell-session
 wasp start
 ```
-to have Wasp download and install new dependency (that happens on start of `wasp start`).
+to have Wasp download and install the new dependency. If `wasp start` is already running, Wasp will detect the dependency change, and restart automatically.
```

Comment:
The docs said to start and re-run `wasp start`, but when I followed the tutorial, I found that `wasp start` helpfully detected the dependency change and restarted itself. So I updated the docs to say that!

Reply:
Ah true we added that recently :)!